internal/servers/controller/handlers/roles/role_service_test.go

```diff
@@ -1423,7 +1423,7 @@ func checkEqualGrants(t *testing.T, expected []string, got *pb.Role) {
 		require.NoError(err)
 		assert.Equal(expected[i], got.GrantStrings[i])
 		assert.Equal(expected[i], got.Grants[i].GetRaw())
-		assert.Equal(parsed.CanonicalString(), got.Grants[i].GetCanonical())
+		assert.Equal(v, got.Grants[i].GetCanonical())
```

Comment:
Curious about the need for this change, shouldn't `parsed.CanonicalString()` still equal `got.Grants[i].GetCanonical()` after this change...

Reply:
Yes, but I did it this way so that I could ensure that it failed _without_ the change -- the test being like this is why this wasn't caught before. This way it will also catch a regression.